src/main/kotlin/com/ampnet/userservice/service/impl/UserServiceImpl.kt

```diff
@@ -98,6 +95,17 @@ class UserServiceImpl(
         return user
     }
 
+    private fun disconnectUserInfo(user: User) {
+        user.userInfoUuid?.let {
+            userInfoRepository.findById(it).ifPresent { userInfo ->
+                userInfo.connected = false
+                userInfo.deactivated = true
+                userInfoRepository.save(userInfo)
```

Comment:
This line is redundant. Hibernate flushes updates to the database due to @Transactional annotation on the method calling this private method.